### DISCUSSION TITLE
Confusing output during SSH

### DISCUSSION THREAD

**Discussion by abhijit-hota** [kovidgoyal/kitty#6513] [Q&A] [answered]
Whenever I SSH into my mac (with Kitty installed) from my Linux machine, I see this output:

```sh
/dev/stdin:1: bad pattern: ^[[33mkitty
/Users/<username>/.local/share/kitty-ssh-kitten/kitty/bin/kitty: line 110: printf: write error: Broken pipe
```

The line 110 in question is this:
```sh
printf "Downloading kitty from: \033[32m%s\033[m\n\n" "$url"
```

The connection itself succeeds and the rest of the things seem to work (e.g., `cwd` while duplicating panes, etc.).

I SSH by the recommended approach:
```sh
alias ssh="kitty +kitten ssh"
```

I even tried [copying the `terminfo` files manually](https://sw.kovidgoyal.net/kitty/kittens/ssh/#copying-terminfo-files-manually) to no avail.

Any help here? On a similar note, are there some extra instructions on how to handle SSH between two machines that both have Kitty installed?

**Comment by kovidgoyal** [comment, marked as answer]
You have something in your shell rc files that is invoking kitty remove
it and you will be fine.

**Reply by abhijit-hota** [reply]
Indeed. I had the completions script for the kitty CLI in my `.zshrc`:

```
kitty + complete setup zsh | source /dev/stdin
```

Removing it fixed the issue. Thanks!
